packages/@coorpacademy-components/src/molecule/card-content/index.js

```diff
@@ -54,6 +55,9 @@ const ContentInfo = (
     backgroundColor: progressBarColor,
     width: `${progress * 100}%`
   };
+  const externalContent = isExternalContent(type);
+  const titleColor = externalContent ? '13171a' : whiteColor;
```

Comment:
13171a is not a valid color

Reply:
👍

Reply:
il faut mettre les couleur dans le css c'est pas des couleurs du skin

Reply:
done